pageserver/src/tenant/storage_layer.rs

```diff
@@ -458,6 +458,31 @@ pub enum ValueReconstructResult {
     Missing,
 }
 
+/// Layers contain a hint indicating whether they are likely to be used for reads.  This is a hint rather
```

Comment:
extra space after `reads.` and before `This`?

Reply:
hah, when I was a kid we were trained to type this way :older_man: (https://www.cultofpedagogy.com/two-spaces-after-period/)